crates/bevy_type_registry/Cargo.toml

```diff
@@ -9,9 +9,13 @@ repository = "https://github.com/bevyengine/bevy"
 license = "MIT"
 keywords = ["bevy"]
 
+[features]
+default = ["dynamic_plugins"]
+dynamic_plugins = ["bevy_app/dynamic_plugins"]
+
 [dependencies]
 # bevy
-bevy_app = { path = "../bevy_app", version = "0.1" }
+bevy_app = { path = "../bevy_app", version = "0.1", default_features = false }
```

Comment:
I believe the `default` features could be removed, and moved up to the bevy lib itself. Then you wouldn't have to disable default_features in each crate referencing it, which would make it difficult to integrate any future features in

Reply:
Fair point, I guess I was trying too hard to not break anything 😅